src/services/services.ts

```diff
@@ -3839,7 +3837,9 @@ module ts {
                 if (symbolKind) {
                     pushTypePart(symbolKind);
                     displayParts.push(spacePart());
-                    addFullSymbolName(symbol);
+                    if (!symbol.isAnonymous) {
+                        addFullSymbolName(symbol);
+                    }

```

Comment:
@mhegazy here is where I build the display part and I use `.isAnonymous` property to decide whether to add a symbol name. I did a lot of checks before to pin point anonymous declarations. Do you happen to know an effective way?